tortoise/tortoise.go

```diff
@@ -356,6 +356,10 @@ func (t *turtle) encodeVotes(
 				votes.Support = append(votes.Support, bid)
 			case against:
 				votes.Against = append(votes.Against, bid)
+			case abstain:
```

Comment:
maybe instead comment that tortoise cannot vote abstain past hdist.

Reply:
will change to an error, just in case